HW_Sem1/Sem_task6.py

```diff
@@ -0,0 +1,18 @@
+YEAR_CONST1 = 4
+YEAR_CONST2 = 100
+GREG_CALENDAR = 1582
+
+result = None
+
+year = int(input('Напишите год для проверки на високосность: '))
+if year < GREG_CALENDAR:
+    result = 'Грегорианский календарь был введен только в феврале 1582г.'
+elif year % YEAR_CONST1 != 0:
+    result = 'Обычный год'
+elif year % YEAR_CONST2 == 0 and year % (YEAR_CONST1 * YEAR_CONST2) == 0:
```

Comment:
В такой постановке первое условие не нужно
```suggestion
elif year % (YEAR_CONST1 * YEAR_CONST2) == 0:
```